quora-dark.user.css

```diff
@@ -149,9 +149,23 @@
     color: #ddd !important;
     background-color: #333 !important;
   }
-  .qu-bg--white, .qu-bg--white__ignore_dark_mode, .qu-bg--gray_ultralight {
+  .qu-bg--white, .qu-bg--white__ignore_dark_mode, .qu-bg--blue_ultralight {
+    background-color: #363636;
+  }
+  .qu-bg--white, .qu-bg--white__ignore_dark_mode, .qu-bg--gray_ultralight,
+  li.tribes_nav_menu div {
     background-color: #282828;
   }
+  .q-platform--desktop .qu-hover--bg--blue_light:hover {
+    background-color: #444 !important;
+  }
+  .q-platform--desktop .qu-hover--bg--gray_ultralight:hover {
+    background-color: #333 !important;
+  }
+	li.tribes_nav_menu a:hover {
+		color: #ddd;
+    background-color: rgba(50, 50, 50, .7) !important;
+	}
```

Comment:
These seems like the same commits in https://github.com/StylishThemes/Quora-Dark/pull/37/files which will cause a conflict.

can you remove the duplicates please and also rebase on master.

 thanks.

Reply:
Ok, I rebased this branch on master and resolved the conflicts.
I agree with the fix in 5777037. It was my bad, blue shade is only needed for `--blue_ultralight`.